pt-BR.json

```diff
@@ -46,7 +46,7 @@
 	"LEAD_ACTORS": "Atores Principais",
 	"CAST": "Elenco",
 	"CREW": "Equipe técnica",
-	"SHOW_MORE_CAST": "Mostrar mais »",
+	"SHOW_MORE_CAST": "Mostrar mais elenco»",
```

Comment:
this was correct before your change, the actual text in english is also just "show more":
https://github.com/Stremio/stremio-translations/blob/master/en-US.json#L58

Reply:
@raemarchetti can you revert this change too?